### PR TITLE
updating events

### PR DESCRIPTION
**Pull request by jendefig**
Event update.  Also changed the text in the index.tsx to just direct to subscribe, instead of saying "No Upcoming events."  Lmk if this was not the right way to do this.

### CODE REVIEW

File: content/community.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     },
     "events": {
       "title": "Events",
-      "description": "Learn more about DVC and meet us in person! We're appearing at conferences, meetups, and talks around the world.",
+      "description": "Learn more about DVC and meet our team virtually! We're appearing at conferences, meetups, and talks around the world.",
```
Comment (shcheklein) [inline content/community.json:18]: make it `or in person`? so that we don't have to change it next time?

File: content/community.json [hunk 2]
```diff
@@ -71,20 +71,12 @@
   ],
   "events": [
     {
-      "url": "https://dssconf.pl/",
```
Comment (shcheklein) [inline content/community.json:74]: I think, no need to remove anything. Just add a new event.

File: content/community.json (hunk 1 above)
Comment (jendefig) [reply content/community.json]: Good idea!

File: content/community.json (hunk 2 above)
Comment (jendefig) [reply content/community.json:74]: Gotcha!  Put it back.

**Review by shcheklein — CHANGES_REQUESTED** [review]
Make image smaller? I think we don't need it to be full size for this also run it through the minifier.

(omg, we really need to something about us saving images into Git :) )

**Comment by jendefig** [comment]
@shcheklein Reduced from a 500x500 pixel image to a 200 x200 pixel image and ran through tiny.png.  I think it should be good now. 👍

**Comment by jendefig** [comment]
> @shcheklein Reduced from a 500x500 pixel image to a 200 x200 pixel image and ran through tiny.png. I think it should be good now. 👍

Actually in my local it looks a bit blurry this small.  Let me know what you think...

**Comment by shcheklein** [comment]
> Actually in my local it looks a bit blurry this small. Let me know what you think...

yes, it should be actually 700x700 (size on the screen x 2 to make it retina displays compatible). So, in this case we should make it even bigger in terms of dimensions and just run through the minifier.

**Review by shcheklein — APPROVED** [review]

**Comment by shcheklein** [comment]
Thanks @jendefig !

**Pull request merged**